source/languages/en/dataplatform/using-dataplatform/configuration/getting-started-with-cache-proxy.md

```diff
@@ -221,7 +221,7 @@ telnet $CACHE_PROXY_HOST $CACHE_PROXY_STATISTICS_PORT
 
 Returns statistic results:
 
-```javascript
+```Json
```

Comment:
I believe this will need to be

> \`\`\`json

if we want the highlighting to work correctly.